Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1783,14 +1583,8 @@ protected virtual void Dispose(bool dispose)
                         Logger.LogError(ex, "Error disposing {Type}", part.GetType().Name);
                     }
                 }
-
-                _userRepository?.Dispose();
```

Comment:
Yes, the dependency injection framework is responsible for disposing it. Since it's registered with `AddSingleton` it will be disposed when the service collection is disposed (i.e. when the application exits): https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-3.1#singleton

Reply:
I hope so, otherwise we will have some broken DBs

Reply:
I confirmed it with the debugger as well